odd-platform-api/src/main/java/com/provectus/oddplatform/repository/DatasetStructureRepositoryImpl.java

```diff
@@ -19,6 +20,7 @@
 
 @Repository
 @RequiredArgsConstructor
+@Slf4j
```

Comment:
there's nothing calling the logger in this class as far as I see

Reply:
Removed